stories/dynamic-widgets.stories.ts

```diff
@@ -14,20 +14,26 @@ storiesOf('Basics/DynamicWidgets', module).add(
     search.addWidgets([
       instantsearch.widgets.EXPERIMENTAL_dynamicWidgets({
         container: dynamicWidgetsContainer,
+        fallbackWidget: ({ attribute, container }) =>
+          instantsearch.widgets.panel<
+            typeof instantsearch.widgets.refinementList
+          >({
```

Comment:
We should be able to use the `RefinementListWidget` type.

Reply:
yes, it's the same